src/tilemaker/working_files/bike_infrastructure.tm2/project.yml

```diff
@@ -9,9 +9,9 @@ bounds:
   - 180
   - 85.0511
 center: 
-  - -111.0677
-  - 40.5554
-  - 12
+  - -111.1784
+  - 40.0162
```

Comment:
This is making the project open for me with everything outside the viewport to the top.